lib/Conversion/TorchOnnxToTorch/DefaultDomainAtoF.cpp

```diff
@@ -1455,10 +1462,7 @@ void mlir::torch::onnx_c::populateDefaultDomainAtoF(
 
         // Otherwise, collapse the left range into a single dimension:
         // torch._prims.collapse(cr, 0, axis - 1)
-        Value axisLess1Const = rewriter.create<Torch::ConstantIntOp>(
```

Comment:
With this CollapseOp removed, it seems like now there is at most one collapse op (for the right-most dimensions). How can this result in a rank-2 tensor, do you not still need to collapse the left-most dimensions?

Reply:
I removed this note realizing it was necessary. From the sample cases I was looking at it appeared to be unneeded. It was restored when addressing test failures.